### PR TITLE
Add optional exec-env flag to show env in execve

### PR DESCRIPTION
**Pull request by yanivagman**

### CODE REVIEW

File: tracee/event_monitor_ebpf.c [hunk 1]
```diff
@@ -758,6 +764,26 @@ static __always_inline int save_argv(submit_buf_t *submit_p, struct pt_regs *ctx
     return 0;
 }
 
+static __always_inline int save_str_arr(submit_buf_t *submit_p, const char __user *const __user *ptr)
```
Comment (itaysk) [inline tracee/event_monitor_ebpf.c]: can you suffix the name with `_to_buf` so it's consistent with other similar functions?

File: tracee/event_monitor_ebpf.c [hunk 2]
```diff
@@ -1114,26 +1140,15 @@ int syscall__execve(struct pt_regs *ctx,
         return 0;
 
     context.eventid = SYS_EXECVE;
-    context.argnum = 2;
+    context.argnum = 3;
```
Comment (itaysk) [inline tracee/event_monitor_ebpf.c]: what if `show_evex_args` is not set and there are just 2 args saved?

File: tracee/event_monitor_ebpf.c (hunk 1 above)
Comment (yanivagman) [reply tracee/event_monitor_ebpf.c]: Will do

File: tracee/event_monitor_ebpf.c (hunk 2 above)
Comment (yanivagman) [reply tracee/event_monitor_ebpf.c]: You are right. I'll fix it

**Pull request merged**